utils/unittest/unittest_test.go

```diff
@@ -12,15 +12,31 @@ func TestCrashTest_NoMessage(t *testing.T) {
 		crash_NoMessage()
 	}
 
-	CrashTest(t, f, "", "TestCrashTest_NoMessage")
+	CrashTest(t, f, "")
 }
 
 // TestCrashTest_ErrorMessage tests that CrashTest() can read standard messages from stdout before a crash.
 func TestCrashTest_ErrorMessage(t *testing.T) {
 	f := func(t *testing.T) {
 		crash_ErrorMessage()
 	}
-	CrashTest(t, f, "about to crash", "TestCrashTest_ErrorMessage")
+	CrashTest(t, f, "about to crash")
+}
+
+// TestCrashTest_ExitCode tests that CrashTest() catches exit codes other than 1.
+func TestCrashTest_ExitCode(t *testing.T) {
+	f := func(t *testing.T) {
+		crash_ExitCode()
+	}
+	CrashTest(t, f, "exit code != 1")
+}
+
+// TestCrashTest_ExitCodeExplicit tests that CrashTest() catches exit codes other than 1 w/ an explicit exit code.
+func TestCrashTest_ExitCodeExplicit(t *testing.T) {
+	f := func(t *testing.T) {
+		crash_ExitCode()
+	}
+	CrashTest(t, f, "exit code != 1", 4, 3, 2)
```

Comment:
Do we need the ability to test for multiple exit code codes at the same time? Why can't we just pass a single expected exit code, instead of having the ability to pass a list of exit codes?
That way, if an exit code changed, our unit test would fail and we would be forced to decide if we should expect the new exit code.

Reply:
This is mostly for the cases where you expect a signal but do not know which specific one you'll get.  Most common example is `SIGSEGV` or `SIGBUS`.  It definitely won't hurt to have an ability to specify multiple codes IMO.

Reply:
OK, sounds good